LazyNecromancer/Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,90 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour
+{
+    Rigidbody2D playerRB;
+    private Vector2 movement;
+
+    public float moveSpeed = 20.0f;
+    public float slowStop = 1.38f;
+
+    private bool isDodging = false;
+    private bool canDodge = true;
+
+    public float dodgeDuration = .25f;
+    public float dodgeSpeed = 45f;
+    public float dodgeCooldown = 1.5f;
+
+    public float dodgeStunTime = 0.25f;
+    public float dodgeStunFactor = 0.25f;
+    
+    float speed;
+
+    private void Start()
+    {
+        playerRB = GetComponent<Rigidbody2D>();
+
+        speed = moveSpeed;
+    }
+
+    private void FixedUpdate()
+    {
+        HandleMovement();
+    }
+
+    private void Update()
+    {
+        if (!isDodging)
+        {
+            float inputX = Input.GetAxisRaw("Horizontal");
+            float inputY = Input.GetAxisRaw("Vertical");
+
+            movement = new Vector2(inputX, inputY).normalized;
+
+            if (Input.GetKeyDown(KeyCode.Space) && canDodge)
```

Comment:
This may need to be switched out to use Unity's input manager.

Reply:
Looks good!